src/transform/build-headers.ts

```diff
@@ -0,0 +1,29 @@
+import { PassThrough, Readable } from 'stream';
+import countBytes from '../lib/countBytes';
+import extractMime from '../lib/extractMime';
+import { Mime } from '../types/Mime';
+
+export interface TransformationHeaders {
+  contentType: Mime;
```

Comment:
we should add application/json to Mime or use another type here, otherwise we cannot use the $report operation.

Reply:
Yes thats true. We could add json and also instead of using probe to extract the mine we can get it from state. Then it should work for json too

Reply:
love it, especially late in the summer